src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -3,7 +3,8 @@
     [dmohs.react :as react]
     [org.broadinstitute.firecloud-ui.common :as common]
     [org.broadinstitute.firecloud-ui.common.icons :as icons]
-    [org.broadinstitute.firecloud-ui.common.style :as style]))
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    ))
```

Comment:
This is the style I prefer for imports (having the close parens on a separate line), so that adding a new import (fairly common) results in a one-line diff rather than two.